gneiss/balances.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import division
+import numpy as np
+from skbio.stats.composition import clr_inv
+from collections import OrderedDict
+
+
+def _balance_basis(tree_node):
```

Comment:
That makes sense if you think is important to test the midpoint values (for example, if they change the final output will not change) and you believe you can reuse that function. So I leave it up to you.